docs/installation.md

```diff
@@ -309,15 +322,8 @@ once in awhile.
 
 ### Homebrew
 
-You can install scalafmt via Homebrew using a custom formula
-
-```sh
-brew install --HEAD olafurpg/scalafmt/scalafmt
-scalafmt --version // should be @STABLE_VERSION@
-
-// to upgrade between releases
-brew upgrade scalafmt
-```
+The recommended way to install the scalafmt command line tool is with
+[Coursier](#coursier), itself available via Homebrew.
```

Comment:
I think it would be nice to have copy-paste friendly instructions for Homebrew users

````md
The recommended way to install the scalafmt command line tool is with
[Coursier](#coursier), itself available via Homebrew.

```sh
brew install coursier/formulas/coursier
coursier install scalafmt
scalafmt --help
```

If necessary, make sure to follow the Coursier instructions for updating
`$PATH` so that the `scalafmt` binary becomes available in your terminal.
````

Reply:
done.